GETTING HELP.md

```diff
@@ -0,0 +1,14 @@
+Getting help with Etcher
+========================
+
+There are various ways you can get support for Etcher if you're having any issue, concern or idea you'd like to share with us.
+
+Gitter
+------
+
+We have a [Gitter chat room](https://gitter.im/resin-io/chat) for Resin.io open to everyone. The room covers Resin.io as a whole, including but not limited to Etcher. Drop us a line there and the Resin.io staff and community users will be happy to assist.
+
+GitHub
+------
+
+Alternatively, you can open a GitHub issue in Etcher's repository by filling [this form](https://github.com/resin-io/etcher/issues/new). Check the [list of open issues](https://github.com/resin-io/etcher/issues) since your question might have been addressed already.
```

Comment:
>Alternatively, you can open a GitHub issue in Etcher's repository by filling [this form](https://github.com/resin-io/etcher/issues/new). Check the [list of open issues](https://github.com/resin-io/etcher/issues) since your question might have been addressed already.

If you encounter an issue or have a suggestion, head on over to Etcher's `[issue tracker](https://github.com/resin-io/etcher/issues)` and if there isn't a ticket covering it, `[create one](https://github.com/resin-io/etcher/issues/new)` :)